content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md

```diff
@@ -97,24 +97,8 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `check.cache_hit` | boolean | Indicates whether Mixer check call hits local cache. | |
 | `quota.cache_hit` | boolean | Indicates whether Mixer quota call hits local cache. | |
 
-## Deprecated attributes
+## Timestamp and duration attributes format
 
-The following attributes have been renamed. We strongly encourage to use the replacement attributes, as the original names will be removed in subsequent releases:
+Timestamp attribute follows RFC 3339 format. When operating with timestamp attributes, you can use `timestamp` function defined in [CEXL](/docs/reference/config/policy-and-telemetry/expression-language/) to convert a textual timestamp in RFC 3339 format into the `TIMESTAMP` type, for example: `request.time | timestamp("2018-01-01T22:08:41+00:00")`, `response.time > timestamp(2020-02-29T00:00:00-08:00)`.
```

Comment:
attribute follows -> attributes are represented in the
can use -> can use the

Does that second use of the timestamp function need quotes?

Reply:
Done.